semantic/src/site/globals/site.overrides

```diff
@@ -22,7 +22,72 @@ a {
 
 a:hover {
   color: @interactiveHigh;
-  opacity: 0.8;
+  opacity: @hoverOpacity;
+}
+
+.center-text {
+  text-align: center;
+}
+
+.alert {
+  color: @attention;
+}
+
+.clickable {
+  cursor: pointer;
+}
+
+.clickable:hover {
+  opacity: @hoverOpacity;
+}
+
+/*--------------------------------
+   Login page
+---------------------------------*/
+
+#login-container {
+  height: 100vh;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  padding: 20px;
+}
+
+#login-box {
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+  max-width: 450px;
+  box-shadow: @basicBoxShadow;
+  border-radius: @borderRadius;
+  padding: 20px;
+}
+
+.flex-centered {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+}
+
+.login-header {
+  margin: 0 !important;
+  text-transform: uppercase !important;
+  letter-spacing: @letterSpacingRegular !important;
+}
```

Comment:
I wonder if this needs to be `!important` because it's done here and not on the `headers.overrides` file?

Reply:
Yeah, it is required, as the component files take precedence over the ".site" properties, so these don't work without it.

Reply:
Why not move to `headers.overrides` then?